R/explanation.R

```diff
@@ -20,10 +20,10 @@
 #'
 #' @param seed Positive integer. If \code{NULL} a random seed will be used.
 #'
-#' @param ... Additional arguments passed to \code{explain.empirical}, \code{explain.gaussian} or
-#' \code{explain.copula}.
+#' @param ... Additional arguments passed to \code{\link{prepare_data}}
 #'
 #' @details
+#' TODO: Add information about approach.
```

Comment:
Should be added now. Currently it is impossble to guess what happens when length(approach) != n_features